tour.php

```diff
@@ -519,9 +519,7 @@ function tour_add_admin_menu() {
 function tour_admin_settings() {}
 
 function tour_button_shortcode() {
-	return '<div class="tour-dropdown"><button id="tour-list-trigger">Toggle List</button>
-		<ul id="page-tour-list" style="display: none;"></ul>
-	</div>';
+	return '<ul id="page-tour-list"></ul>';
```

Comment:
Could we render the list items server side also?

Reply:
Yea we could try that. I chose the JS route because it's easy to search through the DOM of current page and see if any tours exist, thereby ensuring that only tours that are available on the current page are listed.

Reply:
Ah, that's a good point. Could we do a progressive enhancement approach? I think we should expose the tours in the server generated HTML but we could then remove them through JS if the tour doesn't exist on the page.